blockchain/state_manager/src/lib.rs

```diff
@@ -50,6 +61,11 @@ where
         let state = StateTree::new_from_root(self.bs.as_ref(), state_cid).map_err(Error::State)?;
         state.get_actor(addr).map_err(Error::State)
     }
+
+    pub fn get_bs(&self) -> Arc<DB> {
```

Comment:
maybe could be better named? I don't have an opinion on this but maybe for consistency something like `blockstore` or `get_blockstore`? Some other similar functions use the name `store` but might be better to be more explicit of what it actually is

Reply:
Oh yeah this was silly of me, will change that asap